test/e2e/features/pageObjects/loginLogoutObjects.js

```diff
@@ -13,6 +13,15 @@ function loginLogoutObjects() {
   this.failure_error_heading = element(by.css("[id='validation-error-summary-heading']"));
   this.dashboard_header= element(by.xpath("//a[@class='hmcts-header__link']"));
 
+  this.isLoginPageDisplayed = async function(){
+    try{
+      await await BrowserWaits.waitForElement(this.emailAddress);
```

Comment:
await await?

Reply:
@sreekanthpuligadda I believe this your change

Reply:
updated